dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/datasource/SpringConnectionFactory.java

```diff
@@ -152,7 +152,7 @@ private void initializeH2Datasource() {
                     .setType(EmbeddedDatabaseType.H2)
                     .setScriptEncoding(Constants.UTF_8)
                     .setName("dolphinscheduler;MODE=MySQL;DATABASE_TO_LOWER=TRUE;DB_CLOSE_DELAY=-1")
-                    .addScript(PropertyUtils.getString("spring.datasource.sql.schema", "file:../sql/dolphinscheduler_h2.sql"))
+                    .addScript(PropertyUtils.getString("spring.datasource.sql.schema", "classpath:sql/dolphinscheduler_h2.sql"))
```

Comment:
@ruanwenjun this file path is wrong in local tests and standalone server and the tests are not run in h2 server, I've fixed them in this PR

Reply:
Ok, but this is strange, it can work well on my environment, both test and standalone are use h2.

I agree with move the h2 file to classpath, this is good job.

Reply:
> Ok, but this is strange, it can work well on my environment, both test and standalone are use h2.
> 
> I agree with move the h2 file to classpath, this is good job.

That is wired and looks unreasonable, it should not work as you can see the relative path is wrong.

Reply:
Anyway, let's move them to class path